### ISSUE TITLE
Color is a deal breaker

### ISSUE THREAD

**Issue by rounakcodes** [h-sifat/productivity-timer#3]
I guess this needs to be highlighted in bold or something like a warning icon at the beginning. 

1. The app shows the background color of my terminal!

The library I'm using to build the TUI is called blessed. It's a very old library and it hasn't been updated in the last seven years. It doesn't support inheritance for background color styling. So to fix this I've to manually assign a background color to every element. Even if I assign a value of "#000" it doesn't become as black as my terminal background. So yeah, it's a feature!

**Comment by h-sifat** [comment]
Yeah I know, it sucks! But what do you mean by **at the beginning**? At the top of "Readme" or "Known Issues"?

**Comment by rounakcodes** [comment]
I meant to say that I installed and everything and then I checked the "known issues". It would be nice if this issue **stood out** as it is not something that can be ignored. It is not possible to use the app unless some efforts are taken by the user to fix the colors (at least that's how it was for me with the default colors). So, whatever you can do to highlight it a bit more would save everyone some trouble. Thanks.

**Comment by h-sifat** [comment]
I think rather than highlighting the issue (without providing a solution) I'll start working on making the TUI theme-able. It'll probably take some time. Please be patient and sorry for the inconvenience.
